plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -313,7 +316,8 @@ private ReaderPageSource createParquetPageSource(Path path)
                 parquetDateTimeZone,
                 new FileFormatDataSourceStats(),
                 new ParquetReaderOptions(),
```

Comment:
If we want to set default value to true and also making sure deltaLake has it disabled, bloom filter should be explicitly disabled here.